samples/ClientCredentialsFlow/AuthorizationServer/Startup.cs

```diff
@@ -22,7 +22,7 @@ public Startup(IConfiguration configuration)
 
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddMvc();
+            services.AddMvc(option => option.EnableEndpointRouting = false);
```

Comment:
I must confess I'm not a huge fan of the new endpoint-based routing (it seems a bit... half-baked?), but it's the norm in ASP.NET Core 3.x, so we should update the samples to use it.